website/databend/docusaurus.config.js

```diff
@@ -63,7 +67,18 @@ const config = {
                 sidebarPath: require.resolve('./docs/dev/sidebars.js'),
                 editUrl: 'https://github.com/datafuselabs/databend/edit/main/website/databend',
             },
-        ]
+        ],
+        [
+          '@docusaurus/plugin-client-redirects',
+          {
+            redirects: [
+              {
+                to: '/', // string
+                from: '/overview/building-and-running', // string | string[]
```

Comment:
Please leave a comment for this change so that we can remove it in the future.

Reply:
This is only a temporary solution, this will be removed later when db.io is updated. Consulted with BingXi

Reply:
ok, Let me add a comment, wait a moment